scaling.adoc

```diff
@@ -104,74 +104,74 @@ approaches have since been implemented.
 === Scaling approaches
 
 As hinted above, scaling Bitcoin doesn't necessarily have to be about
-increasing the block size limit or other limits. We'll go through some
-general scaling approaches, of which some don't suffer from the
+increasing the block size limit or other limits. We now go through some
+general approaches to scaling, some of which don't suffer from the
 throughput-decentralization trade-off mentioned in the previous
 section.
 
 ==== Vertical scaling
 
 Vertical scaling is the process of increasing the computing resources
-of the machines processing data. With Bitcoin, this would be
-the full node, the machine that verifies the blockchain on behalf of
-its user.
+of the machines processing data. In the context of Bitcoin, these latter would be
+the full nodes, namely the machines that validate the blockchain on behalf of
+their users.
 
-Typically, vertical scaling in Bitcoin would be an increase of the
+Typically [comm: what do you mean by "typically"? To me it suggests the idea that vertical scaling usually occurs with an increase in the block size, but I don't think this is what you mean], vertical scaling in Bitcoin would be an increase of the
 block size limit. This would require some full nodes to upgrade their
 hardware to keep up with the increasing computational
-demands. The downside is that it happens at the cost of
+demands [comm: I'm not entirely convinced by this line of reasoning; if you increase the block size limit thus the computational demand, you can either have full nodes upgrading their hardwares or just use many more of them, can't you?]. The downside is that it happens at the cost of
```

Comment:
"either have full nodes upgrading their hardwares" -> Yes
"use many more of them" -> No

Each full node must be able to verify everything on their own. That's why it's called a "full" node: It fully verifies every aspect of the blockchain. The owner of the full node can then be sure that his copy of the blockchain is correct. Since the computational requirements to verify the blockchain increases by a block size limit increase, so does the full node owner's cost of running the full node. It becomes more expensive to use Bitcoin in a trustless way.

I hope this made it clearer.

Reply:
Sure, now everything is clear, thank you very much for the explanation. I'm not entirely sure what I was thinking about when I made the comment, I surely wasn't thinking of full nodes.